### ISSUE TITLE
Custom labels are not working on time range picker.

### ISSUE THREAD

**Issue by davidszanto** [Wikiki/bulma-calendar#254]
labelFrom and labelTo are not shown in inline timerange-picker!

There should be a custom label option in inline picker, but even in the demo it is not shown:
https://demo.creativebulma.net/components/calendar/v6/demonstration/time/#range-selection

My configuration:
``` js
this.timePicker = bulmaCalendar.attach('[type="time"]', {
      start: this.defaultStart,
      end: this.defaultEnd,
      minuteSteps: 10,
      showFooter: false,
      displayMode: 'inline',
      labelFrom: 'From',
      labelTo: 'To'
    });
```

**Comment by davidszanto** [comment]
Workaround:
``` js
 const timePicker = bulmaCalendar.attach('[type="time"]', {
      minuteSteps: 10,
      showFooter: false,
      displayMode: 'inline',
      labelFrom: 'From',
      labelTo: 'To'
    });

const element = timePicker[0]._ui.wrapper;
const header = element.getElementsByClassName('datetimepicker-header')[0];
header.style.flexDirection = 'row';
header.classList.remove('is-hidden');
element.getElementsByClassName('datetimepicker-selection-start')[0].classList.add('is-hidden');
element.getElementsByClassName('datetimepicker-selection-end')[0].classList.add('is-hidden');
```
![image](https://user-images.githubusercontent.com/10241958/88391732-0aae6080-cdbb-11ea-851f-3abaada5d2d1.png)

**Comment by michael-hack** [comment]
Fixed with 6.1.2
